src/screens/Learn/Learn.style.tsx

```diff
@@ -45,7 +33,50 @@ export const PageHeader = styled.h1`
   }
 `;
 
-export const PageIntro = styled(MarkdownContent)`
+export const SectionHeader = styled.h2`
+  font-weight: 900;
+  font-size: 1.5rem;
+  line-height: 1.25;
+  margin: 1.25rem 0;
+`;
+
+export const BreadcrumbsContainer = styled.div`
+  margin-bottom: ${theme.spacing(4)}px;
+`;
+
+/*
+  Markdown styles used throughout Learn
+*/
+
+export const BodyCopyStyles = css`
+  color: ${COLOR_MAP.GRAY_BODY_COPY};
+  line-height: 1.4;
+  letter-spacing: 1;
+
+  p {
+    &:not(:last-child) {
+      margin-bottom: 1rem;
+    }
+  }
+`;
+
+const BlockquoteStyles = css`
+  background-color: #fafafa;
+  padding: 1.125rem;
```

Comment:
Consider using spacing from the theme (either rounding to 16 or 24). I'm actually not sure what's best here, using `rem` or theme spacing 🤔

Reply:
the mocks had 20px of spacing so i kept it exact, but i can round down to 16 and use theme spacing